microraiden/microraiden/test/test_examples.py

```diff
@@ -13,7 +14,10 @@
 import microraiden.client.session
 
 
-@pytest.mark.needs_xorg
+@pytest.mark.skipif(
+    ('TEST_SKIP_XORG' in os.environ) or ('DISPLAY' not in os.environ),
```

Comment:
When would you have `os.environ['DISPLAY']` set?

Reply:
It tests for 1) existence of the tkinter library that's required to display price widget and 2) $DISPLAY that's set for a shells spawned from X
Travis has `tkinter` installed but doesn't export `$DISPLAY`...
Now that I am thinking of it, I'll check the variable in `conftest.py`, better to have it in one place.